lib/Doctrine/ORM/Query/FilterCollection.php

```diff
@@ -177,6 +217,18 @@ public function isEnabled($name)
         return isset($this->enabledFilters[$name]);
     }
 
+    /**
+     * Checks if a filter is disabled.
+     *
+     * @param string $name Name of the filter.
+     *
+     * @return bool True if the filter is disabled, false otherwise.
+     */
+    public function isDisabled($name)
```

Comment:
```suggestion
    public function isDisabled(string $name): bool
```

Please always add native types to new methods.